scripts/issues.py

```
@@ -0,0 +1,323 @@
+# /// script
```

Ruff (0.15.2)

[error] The file is executable but no shebang is present

(EXE002)